src/main/java/pvs/app/service/data/TrelloData.java

```diff
@@ -0,0 +1,60 @@
+package pvs.app.service.data;
+
+import java.util.ArrayList;
+
+public class TrelloData {
+    private final ArrayList<TrelloList> lanes;
+
+    public TrelloData(){
+        this.lanes = new ArrayList<>();
+    }
+
+    public TrelloList createList(String id, String title, String label, int width) {
+        return new TrelloList(id, title, label, width);
+    }
+
+    public void addList(TrelloList trelloList) {
+        this.lanes.add(trelloList);
+    }
+
+    public static class TrelloList {
```

Comment:
Since it may hard to understand what is a `TrelloList` (a lot of Trello ?), so
Suggest rename to `Lane`.